.changeset/blue-colts-film.md

```diff
@@ -0,0 +1,23 @@
+---
+'@astrojs/db': minor
+---
+
+Remove `AstroDbIntegration` type
```

Comment:
Just pointing out that it looks like docs will need updating here for this, too? (I currently have a docs PR for the change to astro core!)

https://docs.astro.build/en/guides/astro-db/#building-astro-db-integrations